<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepthViewWindow</class>
 <widget class="QMainWindow" name="DepthViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DepthView</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/MainIcon.png</normaloff>:/icons/MainIcon.png</iconset>
  </property>
  <widget class="ImageWidget" name="imageWidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuStereo_Mode">
    <property name="title">
     <string>Stereo Mode</string>
    </property>
    <widget class="QMenu" name="menuAnglaph">
     <property name="title">
      <string>Anglaph</string>
     </property>
     <addaction name="actionFull_Color"/>
     <addaction name="actionHalf_Color"/>
     <addaction name="actionGreyscale"/>
    </widget>
    <widget class="QMenu" name="menuSide_By_Side">
     <property name="title">
      <string>Side By Side</string>
     </property>
     <addaction name="actionNo_Mirror"/>
     <addaction name="actionMirror_Left"/>
     <addaction name="actionMirror_Right"/>
     <addaction name="actionMirror_Both"/>
    </widget>
    <widget class="QMenu" name="menuInterlaced">
     <property name="title">
      <string>Interlaced</string>
     </property>
     <addaction name="actionHorizontal"/>
     <addaction name="actionVertical"/>
    </widget>
    <widget class="QMenu" name="menuSingle">
     <property name="title">
      <string>Mono View (No 3D Effect)</string>
     </property>
     <addaction name="actionLeft_Image"/>
     <addaction name="actionRight_Image"/>
    </widget>
    <addaction name="menuAnglaph"/>
    <addaction name="menuSide_By_Side"/>
    <addaction name="menuInterlaced"/>
    <addaction name="actionCheckerboard"/>
    <addaction name="menuSingle"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/silk/16x16/zoom.png</normaloff>:/icons/silk/16x16/zoom.png</iconset>
     </property>
     <addaction name="actionZoomIn"/>
     <addaction name="actionZoomOut"/>
     <addaction name="separator"/>
     <addaction name="actionFit"/>
     <addaction name="actionzoom200"/>
     <addaction name="actionzoom100"/>
     <addaction name="actionzoom50"/>
    </widget>
    <addaction name="actionShow_MenuBar"/>
    <addaction name="actionFullscreen"/>
    <addaction name="menuZoom"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionFirst"/>
    <addaction name="actionLast"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuImage"/>
   <addaction name="menuStereo_Mode"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/folder-image.png</normaloff>:/icons/silk/16x16/folder-image.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/information.png</normaloff>:/icons/silk/16x16/information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/cross.png</normaloff>:/icons/silk/16x16/cross.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/disk.png</normaloff>:/icons/silk/16x16/disk.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFull_Color">
   <property name="text">
    <string>Full Color (Red/Cyan)</string>
   </property>
  </action>
  <action name="actionHalf_Color">
   <property name="text">
    <string>Half Color (Red/Cyan)</string>
   </property>
  </action>
  <action name="actionGreyscale">
   <property name="text">
    <string>Greyscale (Red/Cyan)</string>
   </property>
  </action>
  <action name="actionShow_StatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Status Bar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionShow_MenuBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Menu Bar</string>
   </property>
   <property name="toolTip">
    <string>Show Menu Bar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/arrow_out.png</normaloff>:/icons/silk/16x16/arrow_out.png</iconset>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/resultset_next.png</normaloff>:/icons/silk/16x16/resultset_next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/resultset_previous.png</normaloff>:/icons/silk/16x16/resultset_previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionNo_Mirror">
   <property name="text">
    <string>No Mirror</string>
   </property>
  </action>
  <action name="actionMirror_Left">
   <property name="text">
    <string>Mirror Left</string>
   </property>
  </action>
  <action name="actionMirror_Right">
   <property name="text">
    <string>Mirror Right</string>
   </property>
  </action>
  <action name="actionMirror_Both">
   <property name="text">
    <string>Mirror Both</string>
   </property>
  </action>
  <action name="actionFit">
   <property name="text">
    <string>Fit</string>
   </property>
   <property name="shortcut">
    <string>0</string>
   </property>
  </action>
  <action name="actionzoom100">
   <property name="text">
    <string>100%</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionzoom50">
   <property name="text">
    <string>50%</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionzoom200">
   <property name="text">
    <string>200%</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="text">
    <string>Vertical</string>
   </property>
  </action>
  <action name="actionHorizontal">
   <property name="text">
    <string>Horizontal</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/wrench.png</normaloff>:/icons/silk/16x16/wrench.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="actionCheckerboard">
   <property name="text">
    <string>Checkerboard</string>
   </property>
  </action>
  <action name="actionLeft_Image">
   <property name="text">
    <string>Left Image</string>
   </property>
  </action>
  <action name="actionRight_Image">
   <property name="text">
    <string>Right Image</string>
   </property>
  </action>
  <action name="actionFirst">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/resultset_first.png</normaloff>:/icons/silk/16x16/resultset_first.png</iconset>
   </property>
   <property name="text">
    <string>First</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/resultset_last.png</normaloff>:/icons/silk/16x16/resultset_last.png</iconset>
   </property>
   <property name="text">
    <string>Last</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/zoom_in.png</normaloff>:/icons/silk/16x16/zoom_in.png</iconset>
   </property>
   <property name="text">
    <string>In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/silk/16x16/zoom_out.png</normaloff>:/icons/silk/16x16/zoom_out.png</iconset>
   </property>
   <property name="text">
    <string>Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header>imagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
